go/client/cmd_wot.go

```diff
@@ -17,6 +17,7 @@ func newCmdWebOfTrust(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Co
 	return cli.Command{
 		Name:         "wot",
 		Usage:        "Web of Trust",
+		Unlisted:     true,
```

Comment:
It will be easier to flip WOT on if all these WOT related `true`s are instead some feature-flag-like var `bool hideWOTCLI`

Reply:
you're right, but then we'd have more code to delete.